### PR TITLE
Game Labs - Video Library App

### PR DESCRIPTION
**Pull request by OmkarArora**
Completed the initial frontend part of the video library app, tried giving it youtube's design and layout.
Live preview - https://game-labs.netlify.app/

### CODE REVIEW

**Review by Harsh-D — APPROVED** [review]
I have gone through the codebase. Looks good to me.
Code has been properly structured. No changes suggested from my side.

**Review by Ayan4 — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      <main className="main">
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/guides" element={<Guides />} />
+          <Route path="/library" element={<Library />} />
+          <Route path="/playlist/:playlistId" element={<ExplorePlaylist />} />
+          <Route path="/video/:videoId" element={<ExploreVideo />} />
+        </Routes>
+      </main>
+      {windowWidth < 768 && (
```
[the remark this answers is not in the document]
Comment (OmkarArora) [reply src/App.jsx:33]: Yup, got it.

**Comment by sdhamani** [comment]
The site looks good. The git commits are impressive. 

The only thing is the comment in the guide context.It is from a different context.

image

Apart from that, I guess you are good to go.

**Comment by OmkarArora** [comment]
> The site looks good. The git commits are impressive.
> 
> The only thing is the comment in the guide context.It is from a different context.
> 
> image
> 
> Apart from that, I guess you are good to go.

Ok got it, I'll update it when I integrate the backend shortly.

**Pull request closed without merging**